python_modules/dagster/dagster_tests/core_tests/execution_tests/test_asset_backfill.py

```diff
@@ -153,7 +153,12 @@ def scenario(
         (
             "partitioned_after_non_partitioned",
             ["2022-01-01", "2022-01-02"],
-            [("asset1", None), ("asset2", "2022-01-01"), ("asset2", "2022-01-02")],
+            [
```

Comment:
note: the old version of this test still would have passed, but I updated the set of assets in partitioned_after_non_partitioned to have two root unpartitioned assets instead of one, so this needed to be changed accordingly